react/src/tests/Emails-test.js

```diff
@@ -0,0 +1,669 @@
+        resending: {
+          is_fetching: false,
+          failed: false,
+          error: {},
+          message: ''
+        },
+        confirming: 'test@localhost.com',
+        emails: [],
+        email: '',
+      }
+    );
+  });
+
+    it("Receives a STOP_CONFIRMATION action", () => {
+    expect(
+      emailsReducer(
+        mockState,
+        {
+          type: actions.STOP_CONFIRMATION,
+        }
+      )
+    ).toEqual(
+      {
+        is_fetching: false,
+        failed: false,
+        error: '',
+        message: '',
+        resending: {
+          is_fetching: false,
+          failed: false,
+          error: {},
+          message: ''
+        },
+        confirming: '',
+        emails: [],
+        email: '',
+      }
+    );
+  });
+
+    it("Receives a START_RESEND_EMAIL_CODE action", () => {
+    expect(
+      emailsReducer(
+        mockState,
+        {
+          type: actions.START_RESEND_EMAIL_CODE,
+        }
+      )
+    ).toEqual(
+      {
+        is_fetching: false,
+        failed: false,
+        error: '',
+        message: '',
+        resending: {
+          is_fetching: true,
+          failed: false,
+          error: {},
+          message: ''
+        },
+        confirming: '',
+        emails: [],
+        email: '',
+      }
+    );
+  });
+
+it("Receives a START_RESEND_EMAIL_CODE_SUCCESS action", () => {
+    expect(
+      emailsReducer(
+        mockState,
+        {
+          type: actions.START_RESEND_EMAIL_CODE_SUCCESS,
+            message: 'emails.resend_success'
+        }
+      )
+    ).toEqual(
+      {
+        is_fetching: false,
+        failed: false,
+        error: '',
+        message: '',
+        resending: {
+          is_fetching: false,
+          failed: false,
+          error: {},
+          message: 'emails.resend_success'
+        },
+        confirming: '',
+        emails: [],
+        email: '',
+      }
+    );
+  });
+
+it("Receives a START_RESEND_EMAIL_CODE_FAIL action", () => {
+    expect(
+      emailsReducer(
+        mockState,
+        {
+          type: actions.START_RESEND_EMAIL_CODE_FAIL,
+           payload: {
+              error: {error:"Bad error"},
+         }
+        }
+      )
+    ).toEqual(
+      {
+        is_fetching: false,
+        failed: false,
+        error: '',
+        message: '',
+        resending: {
+          is_fetching: false,
+          failed: true,
+          error: {error:"Bad error"},
+          message: ''
+        },
+        confirming: '',
+        emails: [],
+        email: '',
+      }
+    );
+  });
+
+});
+
+const getState = () => state;
+const state = {
+    emails: {
+        is_fetching: false,
+        failed: false,
+        error: '',
+        message: '',
+        confirming: '',
+        emails: [],
+        email: '',
+        resending: {
+          is_fetching: false,
+          failed: false,
+          error: {},
+          message: ''
+        },
+    },
+    confirming: '',
```

Comment:
I put confirming key there because I saw that: https://github.com/SUNET/eduid-html/blob/eperez-email-ms/react/src/reducers/Emails.js#L16